app/controllers/answer_like_controller.rb

```diff
@@ -11,8 +11,7 @@ def likes
       AnswerSelection.set_likes(current_user.uid, @answer.id, false)
     else
       AnswerSelection.set_likes(current_user.uid, @answer.id, true)
-      user = User.find(current_user.uid)
-      AnswerMailer.notify_answer_like(user, @answer).deliver_later
+      User.find(current_user.uid)
```

Comment:
I think this is not required. We already have user as `current_user`.
😄

Reply:
Yeah, I think that line didn't make any sense. Thanks for pointing out!!